instance/static/html/instance/index.html

```diff
@@ -24,6 +24,13 @@
       <li ng-repeat="instance in instanceList"
           ng-class="{active: instance == selected.instance}">
         <a ng-click="select('instance', instance)">{{ instance.name }}</a>
+        <div class="status-icon" ng-switch="instance.status">
+          <i class="fa fa-ellipsis-h" tooltip-placement="top" tooltip="Provisioning in progress" ng-switch-default></i>
+        </div>
```

Comment:
It's not just with state=provisioning that this icone would make sense - there is also active, started, etc where it would make sense to show it to the user, otherwise there will be something running, but nothing to show to the user. Imho it makes more sense to try to get to a 1-to-1 matching between the instance progress and the icons we show here for the instance, ie:

* instance.progress=running => fa-ellipsis
* instance.progress=failed => fa-times
* instance.progress=success => fa-check

And then align the instance progress field value to always be at RUNNING when operations are pending or in progress - it's true that it's not consistently the case currently, but I'd rather expose these issues. Checking on individual states would hide them further.